SimpleRss/SimpleRss/Services/XMLParserService.swift

```diff
@@ -8,13 +8,13 @@
 
 import Foundation
 
-class XMLParserService {
-    func fetchXMLData(for url: String?, withCallback completionHandler: @escaping (_ result: [Feed]?, _ error: Error?) -> Void) {
-
+class XMLParserService: NSObject {
+    func fetchXMLData(for link: String, withCallback completionHandler: @escaping (_ result: [Feed]) -> Void) {
```

Comment:
If class doesn't have any properties or internal state, it's probably a good idea to make its funcs `static`